src/components/network-modification-tree-pane.js

```diff
@@ -342,6 +350,23 @@ export const NetworkModificationTreePane = ({
                         removeNotificationByNode([currentNodeRef.current?.id])
                     );
                 }
+                //creating, updating or deleting modifications must invalidate the node clipboard
+            } else if (
+                UPDATE_TYPE.includes(
+                    studyUpdatedForce.eventData.headers['updateType']
+                )
+            ) {
+                if (
+                    (true === isInitiatingCopyTab.current &&
+                        studyUpdatedForce.eventData.headers['parentNode'] ===
+                            selectionForCopyRef.current.nodeId) ||
+                    isSubtreeImpacted('parentNode')
+                ) {
+                    dispatch(setSelectionForCopy(noSelectionForCopy));
```

Comment:
It could be nice to place this code in a function (something like `checkClipboard` or something) to avoid reset ( `dispatch(setSelectionForCopy(noSelectionForCopy))`) and snackbar code duplication everywhere.

Reply:
And do we need to broadcast here (`broadcastChannel.postMessage(noSelectionForCopy);`) ?